tests-perf/locust/locust.conf

```diff
@@ -1,8 +1,8 @@
 # master.conf in current directory
 locustfile = tests-perf/locust/locust-notifications.py
 host = https://api.staging.notification.cdssandbox.xyz
-users = 2500
-spawn-rate = 200
+users = 3000
+spawn-rate = 20
```

Comment:
Do you need to go this far down in spawn-rate? ie what about 100?

Reply:
That would be 6000 new users per minute, a tad aggressive given that staging does not have continual warm up like production can have. We can adjust later and try to break it but for now, I'd want something stable.